bin/kafka-avro-console-consumer

```diff
@@ -16,7 +16,13 @@
 
 base_dir=$(dirname $0)/..
 
-CLASSPATH=$CLASSPATH:$base_dir/share/java/avro-serializer/*
+# Production jars
+export CLASSPATH=$CLASSPATH:$base_dir/share/java/kafka-serde-tools/*
+
+# Development jars. `mvn package` should collect all the required dependency jars here
```

Comment:
Why not add add at least the development stuff to classpath in `schema-registry-run-class`?